noWord/common/DefaultTransformation.py

```diff
@@ -0,0 +1,74 @@
+
+
+def merge( input, params, context):
+
+    targetResource = {}
+
+    if isinstance( input, list):
+        for resourceName in block['resources']:
+            data = context.getResource(context.resources, resourceName)
+            targetResource.update(deepcopy(data))

```

Comment:
deepcopy is not imported from copy package

Reply:
Ok

Reply:
DONE